.github/workflows/push-master-environment.yaml

```diff
@@ -43,6 +43,7 @@ jobs:
           NAMESPACE: mainnet-eosrate
           # webapp
           REACT_APP_RATING_CONTRACT: rateproducer
+          REACT_APP_MAINNET_VERSION: ${{ secrets.REACT_APP_MAINNET_VERSION }}
```

Comment:
this dosen't have to be a secret the value can be included directly in this file

Reply:
I thought it should be a secret because if we are in the testnet environment it should have value and if we are in the mainnet environment it should have another one and if we burn it in that file we should edit it before merging to master.